src/@next/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Button, PrimaryButton } from '../Button';
+import { Neutral } from '../utilities/colors';
+import {
+  EmptyStateContainer,
+  EmptyStateContentContainer,
+  StyledButtonGroup,
+  StyledDescription,
+  StyledHelpText,
+  StyledImage,
+  StyledTitle,
+} from './EmptyStateStyle';
+
+export enum ImageName {
+  EMPTY_CARTON = 'empty-carton',
+  EMPTY_MAILBOX = 'empty-mailbox',
+  SAFETY_CONE = 'safety-cone',
+}
+
+export interface EmptyStateProps {
+  buttonBasicContent?: React.ReactNode;
+  buttonPrimaryContent?: React.ReactNode;
```

Comment:
we would need callbacks (onClick) props? from what i can see these contents are added within the Button itself

Reply:
@Afsal102 wouldn't you be able to achieve that passing the whole component with onClick event handler attached?

But  @ninariccimarie , I think to strictly enforce only primary button and secondary buttons are rendered, we should accept label and event for those props and render the primary and basic in the component directly

Reply:
thanks for the catch @Afsal102